src/App.tsx

```diff
@@ -1,19 +1,70 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { FilterBy } from './types/FilterBy';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ALL);
+  const [errorStatus, setErrorStatus] = useState(false);
```

Comment:
would be great to move logic from App, component
It isn't required but it's better to keep App as simple as possible